### PR TITLE
created partners-page-ui

### PR DESCRIPTION
**Pull request by mageexcel**

### CODE REVIEW

**Comment by Skraye** [comment]
image
Card isnt animated, also for the same card in the Service partners

**Comment by Skraye** [comment]
image
Wrong line height ( should be 22px)

**Comment by Skraye** [comment]
image
Please make the page accessible from the resources menu (under Community)

File: public/partners/integrate-with-logos/logo-horizontal-color.svg
Comment (Skraye) [inline public/partners/integrate-with-logos/logo-horizontal-color.svg:1]: Please name the file as "tabular.svg"

**Comment by mageexcel** [comment]
@Skraye Can you tell me what should the description for the partners page in resource menu
image

File: components/partners/BecomeAPartnerForm.vue [hunk 1]
```diff
@@ -0,0 +1,99 @@
+<template>
+    <div class="card">
+        <div class="text-dark p-2">
+            <div class="card-body">
+                <form>
+                    <div class="row mb-3">
```
Comment (Skraye) [inline components/partners/BecomeAPartnerForm.vue:6]: Please make it work like here: https://github.com/kestra-io/kestra.io/blob/main/components/contact/Header.vue#L68-L68

**Comment by Skraye** [comment]
> @Skraye Can you tell me what should the description for the partners page in resource menu image

You can use "Elevate your Kestra use through our partner ecosystem"

File: components/partners/Partners.vue [hunk 2]
```diff
@@ -0,0 +1,123 @@
+<template>
+    <div class="container">
+        <Section title="Technologies Partners">
```
Comment (anna-geller) [inline components/partners/Partners.vue:3]:
```suggestion
        <Section title="Technology Partners">
```

File: components/partners/Partners.vue [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+                            <h5 class="card-title">Databricks</h5>
+                            <div class="coming-soon d-inline-block rounded text-white">Coming soon...</div>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-12 col-md-4 col-lg-4 mb-4">
+                    <div class="card text-center" data-aos="zoom-in">
+                        <img src="/partners/Badge_Kestra_Technology_Partner.svg" class="card-img-top" alt="Badge_Kestra_Technology_Partner" height="118" width="118">
+                        <div class="card-body">
+                            <h5 class="card-title mb-3">Want to be listed as a Kestra Partner?</h5>
```
Comment (anna-geller) [inline components/partners/Partners.vue:49]:
```suggestion
                            <h5 class="card-title mb-3">Want to join our Partner Program?</h5>
```

File: components/partners/Partners.vue [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+                </div>
+            </div>
+        </Section>
+        <Section title="Services Partners">
+            <div class="row card-group mb-2">
+                <div class="col-12 col-md-4 col-lg-4 mb-4">
+                    <div class="card" data-aos="zoom-in">
+                        <img src="/partners/partners-logos/converteologo.svg" class="card-img-top" alt="converteologo">
+                        <div class="card-body">
+                            <p class="card-text">Service Partner</p>
```
Comment (anna-geller) [inline components/partners/Partners.vue:62]:
```suggestion
                            <p class="card-text">Consulting Partner</p>
```

File: components/partners/Partners.vue [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+                            <h5 class="card-title">Converteo</h5>
+                            <a href="#">Visit the Converteo website <OpenInNew /></a>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-12 col-md-4 col-lg-4 mb-4">
+                    <div class="card" data-aos="zoom-in">
+                        <img src="/partners/partners-logos/nticologo.svg" class="card-img-top" alt="nticologo">
+                        <div class="card-body">
+                            <p class="card-text">Service Partner</p>
```
Comment (anna-geller) [inline components/partners/Partners.vue:72]:
```suggestion
                            <p class="card-text">Consulting Partner</p>
```

File: components/partners/Partners.vue [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+                        <img src="/partners/Badge_Kestra_Technology_Partner.svg" class="card-img-top" alt="Badge_Kestra_Technology_Partner" height="118" width="118">
+                        <div class="card-body">
+                            <h5 class="card-title mb-3">Want to be listed as a Kestra Partner?</h5>
+                            <button class="btn btn-primary">Become a Technology Partner</button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </Section>
+        <Section title="Services Partners">
```
Comment (anna-geller) [inline components/partners/Partners.vue:56]:
```suggestion
        <Section title="Consulting Partners">
```

File: components/partners/Partners.vue [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+                            <h5 class="card-title">Ntico</h5>
+                            <div class="coming-soon d-inline-block rounded text-white">Coming soon...</div>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-12 col-md-4 col-lg-4 mb-4">
+                    <div class="card text-center" data-aos="zoom-in">
+                        <img src="/partners/Badge_Kestra_Technology_Partner.svg" class="card-img-top" alt="Badge_Kestra_Technology_Partner" height="118" width="118">
+                        <div class="card-body">
+                            <h5 class="card-title mb-3">Want to be listed as a Kestra Partner?</h5>
```
Comment (anna-geller) [inline components/partners/Partners.vue:82]:
```suggestion
                            <h5 class="card-title mb-3">Want to join our Partner Program?</h5>
```

**Comment by tchiotludo** [comment]
here is the latest error: 
```
 Could not resolve "../../public/partners/Frame%20922.svg" from "components/partners/Headers.vue"
```

**Pull request merged**